### DISCUSSION TITLE
Welcome to EPANET2.2 Discussions!

### DISCUSSION THREAD

**Discussion by cbuahin** [USEPA/EPANET2.2#67] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Frecooz** [comment]
Hi. I'm wondering if someone has already build a Linux deb package for EPANET 2.2. I'm not an IT, I spend 6 hours trying and failed miserably.

**Reply by cbuahin** [reply]
Good question, @Frecooz.  I have no knowledge of someone building a deb package for EPANET. However, you can easily modify the CPACK command in the cmake file to generate a [deb package](https://cmake.org/cmake/help/latest/cpack_gen/deb.html).  @woohn and @ucchejbb, perhaps this is something we can configure in future releases.

Also, please consider opening a new discussion for new items in the future as it makes it easier to track different discussion items separately.

**Reply by ucchejbb** [reply]
@cbuahin This would be a good topic for our EPANET visioning discussion for the future of EPANET - How to deliver EPANET to end users.  We can certainly look into this for future release as you mention.
